scripts/test/Muon/loading_tab/loadwidget_presenter_multiple_file_test.py

```diff
@@ -0,0 +1,143 @@
+import unittest
+
+try:
+    from unittest import mock
+except ImportError:
+    import mock
+
+from PyQt4 import QtGui
+from PyQt4.QtGui import QApplication
+
+from Muon.GUI.MuonAnalysis.load_widget.load_widget_model import LoadWidgetModel
+from Muon.GUI.MuonAnalysis.load_widget.load_widget_view import LoadWidgetView
+from Muon.GUI.MuonAnalysis.load_widget.load_widget_presenter import LoadWidgetPresenter
+
+from Muon.GUI.MuonAnalysis.loadrun.load_run_model import LoadRunWidgetModel
+from Muon.GUI.MuonAnalysis.loadrun.load_run_presenter import LoadRunWidgetPresenter
+from Muon.GUI.MuonAnalysis.loadrun.load_run_view import LoadRunWidgetView
+
+from Muon.GUI.MuonAnalysis.loadfile.load_file_model import BrowseFileWidgetModel
+from Muon.GUI.MuonAnalysis.loadfile.load_file_presenter import BrowseFileWidgetPresenter
+from Muon.GUI.MuonAnalysis.loadfile.load_file_view import BrowseFileWidgetView
+
+from Muon.GUI.Common.muon_load_data import MuonLoadData
+
+# global QApplication (get errors if > 1 instance in the code)
+QT_APP = QApplication([])
+
+
+class LoadRunWidgetPresenterMultipleFileTest(unittest.TestCase):
+    class Runner:
```

Comment:
did we split the runner out in a different PR?

Reply:
Yes we did I have now updated this as well